lib/proxy.ts

```diff
@@ -49,21 +49,25 @@ interface CreateVaultResponse {
 
 export async function createVaultProxy(
   vaultFactory: VaultFactory,
+  _admin: HardhatEthersSigner,
   _owner: HardhatEthersSigner,
   _operator: HardhatEthersSigner,
+  initializationParams: Partial<DelegationInitializationParamsStruct> = {},
 ): Promise<CreateVaultResponse> {
   // Define the parameters for the struct
-  const initializationParams: DelegationInitializationParamsStruct = {
-    curatorFeeBP: 100n,
-    nodeOperatorFeeBP: 200n,
+  const defaultParams: DelegationInitializationParamsStruct = {
+    defaultAdmin: await _admin.getAddress(),
     curator: await _owner.getAddress(),
     funderWithdrawer: await _owner.getAddress(),
     minterBurner: await _owner.getAddress(),
     nodeOperatorManager: await _operator.getAddress(),
     nodeOperatorFeeClaimer: await _owner.getAddress(),
+    curatorFeeBP: 100n,
+    nodeOperatorFeeBP: 200n,
```

Comment:
No sure about those hardcoded defaults, to be honest. Can we make it mandatory to ask them at the `createVaultProxy` level to prevent missing / forgetting about them during some deployments?

Reply:
agree, done